### ISSUE TITLE
v0.3.0 libstdc++.so.6: version `GLIBCXX_3.4.26' not found on Raspberry Pi 

### ISSUE THREAD

**Issue by obalyuk87** [mpromonet/webrtc-streamer#343]
**Describe the bug**
Published version v0.3.0 does not run on Raspberry Pi. Older versions work as expected.

Crashes with error message 
```
 ./webrtc-streamer: /usr/lib/arm-linux-gnueabihf/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by ./webrtc-streamer)
```

Probably related to: https://github.com/mpromonet/webrtc-streamer/issues/258

**To Reproduce**
* Option 1: run docker version on  Raspberry Pi 
  `FROM mpromonet/webrtc-streamer:v0.3.0-armv7`... fails
   Note that `mpromonet/webrtc-streamer:v0.2.7-armv7` works fine

* Option 2: download armv7 static binary from releases and try to run binary.
![image](https://user-images.githubusercontent.com/7018369/89726468-e96b8800-d9e8-11ea-9f90-f9d14797b0cf.png)
```
 cd /tmp/
wget https://github.com/mpromonet/webrtc-streamer/releases/download/v0.3.0/webrtc-streamer-v0.3.0-Linux-armv7l-Release.tar.gz
tar xzfv webrtc-streamer-v0.3.0-Linux-armv7l-Release.tar.gz
cd webrtc-streamer-v0.3.0-Linux-armv7l-Release/
./webrtc-streamer
```

**Expected behavior**
should work without error :-)

**Desktop (please complete the following information):**
 - Raspbian GNU/Linux 10 (buster)
 
**Additional context**
AMAZING PROJECT! Thank you!

**Comment by mpromonet** [comment]
Hi obalyuk87 ,

You are right I recently remove the fix introduce for #258 due to conflict between CXX lib use by WebRTC SDK and webrtc-streamer code.
However this should not affect running with docker (that contains a enouph recent GLIBCXX).

Best Regards,
Michel.

**Comment by mpromonet** [comment]
Hi obalyuk87 ,

It should works back with latest docker.

Best Regards,
Michel.
